Druid build fails with git command '-no-show-signature'

Building Druid with the below command on master/v27.0 branches:
```
mvn clean package  install -DskipTests   -Pdist -Prat
````

the build  failed with the below error.

```
[ERROR] Failed to execute goal pl.project13.maven:git-commit-id-plugin:4.9.10:revision (default) on project druid: Could not complete Mojo execution...: pl.project13.core.NativeGitProvider$NativeCommandException: Git command exited with invalid status [128]: directory: `/root/bassa/apache/druid`, command: `git log -1 --pretty=format:%ct --no-show-signature HEAD`, stdout: ``, stderr: `fatal: unrecognized argument: --no-show-signature`
```

Solution:
comment the plugin from https://github.com/apache/druid/blob/master/pom.xml#L1569 to https://github.com/apache/druid/blob/master/pom.xml#L1607

Ex: 
```
            <!--plugin>
                <groupId>pl.project13.maven</groupId>
                <artifactId>git-commit-id-plugin</artifactId>
                <version>4.9.10</version>
                <executions>
                    <execution>
                        <goals>
                            <goal>revision</goal>
                        </goals>
                    </execution>
                </executions>
                <configuration>
                    <dotGitDirectory>${project.basedir}/.git</dotGitDirectory>
                    <dateFormatTimeZone>Etc/UTC</dateFormatTimeZone>
                    <skipPoms>false</skipPoms>
                    <format>json</format>
                    <generateGitPropertiesFile>true</generateGitPropertiesFile>
                    <generateGitPropertiesFilename>${project.build.directory}/git.version</generateGitPropertiesFilename>
                    <failOnNoGitDirectory>false</failOnNoGitDirectory>
                    <useNativeGit>true</useNativeGit>
                    <excludeProperties>
                        <excludeProperty>git.build.user.email</excludeProperty>
                        <excludeProperty>git.build.host</excludeProperty>
                        <excludeProperty>git.commit.id.describe-short</excludeProperty>
                        <excludeProperty>git.commit.user.*</excludeProperty>
                        <excludeProperty>git.commit.message.*</excludeProperty>
                        <excludeProperty>git.closest.tag.*</excludeProperty>
                        <excludeProperty>git.commit.id.abbrev</excludeProperty>
                        <excludeProperty>git.dirty</excludeProperty>
                    </excludeProperties>
                    <gitDescribe>
                        <skip>false</skip>
                        <always>true</always>
                        <abbrev>7</abbrev>
                        <dirty>-dirty</dirty>
                        <forceLongFormat>true</forceLongFormat>
                    </gitDescribe>
                </configuration>
            </plugin-->

```

After commenting the above line, build crosses this error stage.

--- a/codestyle/checkstyle-suppressions.xml
+++ b/codestyle/checkstyle-suppressions.xml
@@ -24,54 +24,13 @@
     "https://checkstyle.org/dtds/suppressions_1_2.dtd">
 
 <suppressions>
-  <!-- See http://checkstyle.sourceforge.net/config_filters.html#SuppressionFilter for examples -->
+  <!-- See https://checkstyle.org/filters/suppressionfilter.html for examples -->
 
   <!-- Code copied from TestNG to apply a bugfix -->
   <suppress checks="AvoidStaticImport" files="[\\/]org[\\/]testng[\\/]" />
 
-  <suppress checks="LocalFinalVariableName" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="SingleSpaceSeparator" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="LocalVariableName" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="AvoidStaticImport" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="ImportOrder" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="RedundantImport" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="UnusedImports" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="EmptyStatement" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="Regexp" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="WhitespaceAfter" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="ModifierOrder" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="RedundantModifier" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="OneStatementPerLine" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="RightCurly" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="MethodName" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="NeedBraces" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="WhitespaceAround" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="Header" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="EmptyLineSeparator" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="LeftCurly" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="Indentation" files="[\\/]target[\\/]generated-sources[\\/]" />
-  <suppress checks="ConstantName" files="[\\/]target[\\/]generated-sources[\\/]" />
-
-  <suppress checks="Header" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="NeedBraces" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="UnusedImports" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="AvoidStarImport" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="FileTabCharacter" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="ConstantName" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="WhitespaceAfter" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="WhitespaceAround" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="ParenPad" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="SingleSpaceSeparator" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="NoWhitespaceBefore" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="NoWhitespaceAfter" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="LeftCurly" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="RightCurly" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="Indentation" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="ModifierOrder" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="RedundantImport" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="OneStatementPerLine" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="ImportOrder" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="EmptyLineSeparator" files="[\\/]target[\\/]generated-test-sources[\\/]" />
+  <suppress checks=".*" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks=".*" files="[\\/]target[\\/]generated-test-sources[\\/]" />
 
   <suppress checks="Indentation" files="ProtoTestEventWrapper.java" />
   <suppress checks="Regexp" id="argumentLineBreaking" files="ProtoTestEventWrapper.java" />

--- a/codestyle/checkstyle-suppressions.xml
+++ b/codestyle/checkstyle-suppressions.xml
@@ -29,23 +29,50 @@
   <!-- Code copied from TestNG to apply a bugfix -->
   <suppress checks="AvoidStaticImport" files="[\\/]org[\\/]testng[\\/]" />
 
+  <suppress checks="LocalFinalVariableName" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="SingleSpaceSeparator" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="LocalVariableName" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="AvoidStaticImport" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="ImportOrder" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="RedundantImport" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="UnusedImports" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="EmptyStatement" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="Regexp" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="WhitespaceAfter" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="ModifierOrder" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="RedundantModifier" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="OneStatementPerLine" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="RightCurly" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="MethodName" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="NeedBraces" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="WhitespaceAround" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="Header" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="EmptyLineSeparator" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="LeftCurly" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="Indentation" files="[\\/]target[\\/]generated-sources[\\/]" />
+  <suppress checks="ConstantName" files="[\\/]target[\\/]generated-sources[\\/]" />
+
   <suppress checks="Header" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="NeedBraces" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="UnusedImports" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="AvoidStarImport" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="FileTabCharacter" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-
+  <suppress checks="ConstantName" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="WhitespaceAfter" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="WhitespaceAround" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="ParenPad" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="SingleSpaceSeparator" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="NoWhitespaceBefore" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="NoWhitespaceAfter" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-
   <suppress checks="LeftCurly" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="RightCurly" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-
   <suppress checks="Indentation" files="[\\/]target[\\/]generated-test-sources[\\/]" />
+  <suppress checks="ModifierOrder" files="[\\/]target[\\/]generated-test-sources[\\/]" />
+  <suppress checks="RedundantImport" files="[\\/]target[\\/]generated-test-sources[\\/]" />
+  <suppress checks="OneStatementPerLine" files="[\\/]target[\\/]generated-test-sources[\\/]" />
+  <suppress checks="ImportOrder" files="[\\/]target[\\/]generated-test-sources[\\/]" />
+  <suppress checks="EmptyLineSeparator" files="[\\/]target[\\/]generated-test-sources[\\/]" />
+
   <suppress checks="Indentation" files="ProtoTestEventWrapper.java" />
   <suppress checks="Regexp" id="argumentLineBreaking" files="ProtoTestEventWrapper.java" />
   <suppress checks="[a-zA-Z0-9]*" files="ProtoTestEventWrapper.java" />
@@ -56,7 +83,6 @@
   <suppress checks="ConstantName" files="Author" />
   <suppress checks="ConstantName" files="Book" />
 
-  <suppress checks="OneStatementPerLine" files="[\\/]target[\\/]generated-test-sources[\\/]" />
 
   <!-- extendedset is a fork of Alessandro Colantonio's CONCISE (COmpressed 'N' Composable Integer SEt) repository
        and licensed to ASF under a CLA is not true. -->
@@ -67,7 +93,5 @@
   <!-- See https://github.com/checkstyle/checkstyle/issues/5510 and the ImportOrder definition in checkstyle.xml -->
   <suppress checks="ImportOrder" message="^'java\..*'.*" />
 
-  <suppress checks="ImportOrder" files="[\\/]target[\\/]generated-test-sources[\\/]" />
-  <suppress checks="EmptyLineSeparator" files="[\\/]target[\\/]generated-test-sources[\\/]" />
   <suppress checks="AvoidStaticImport" files="[\\/]src[\\/](test)[\\/]"/>
 </suppressions>

--- a/pom.xml
+++ b/pom.xml
@@ -2108,9 +2108,14 @@
                                 <exclude>**/test/resources/**/*</exclude> <!-- test data for "old" ITs. -->
                                 <exclude>**/docker/client_tls/*</exclude> <!-- generated files for "old" ITs. -->
                                 <exclude>resources/data/**/*</exclude> <!-- test data for "new" ITs. -->
-                                <exclude>**/derby.log</exclude>
                                 <exclude>**/jvm.config</exclude>
                                 <exclude>**/*.avsc</exclude>
+                                <exclude>**/*.png</exclude>
+                                <exclude>**/*.log</exclude>
+                                <exclude>**/*.jpg</exclude>
+                                <exclude>**/*.gif</exclude>
+                                <exclude>examples/quickstart/protobuf/metrics.desc</exclude>
+                                <exclude>examples/var/sv/.lock</exclude>
                                 <exclude>**/*.iml</exclude>
                                 <exclude>**/*.json</exclude>
                                 <exclude>**/*.parq</exclude>

